torch/utils/_pytree.py

```diff
@@ -72,7 +72,6 @@ def _register_pytree_node(
     typ: Any,
     flatten_fn: FlattenFunc,
     unflatten_fn: UnflattenFunc,
-    *,
```

Comment:
Are we ever able to undo this change?

Reply:
Yes, but whenever torch packages become outdated internally...but I'm not sure when that is. We will remove the call that is passing in positional args in D48784113, so future migration will hopefully be easier...